### ISSUE TITLE
2021 Milestone: 2% complete 

### ISSUE THREAD

**Issue by lukeswitz** [guardianproject/haven#456]
We are all waiting on @n8fr8 to either resurrect or sunset this project. Many items in the Spring 2021 sprint aren’t assigned or assignable, as they are broad ideas without guidance on protocol choices or other important implementations. 

As it stands, it’s non-functional, insecure, outdated, and hasn’t been updated on the primary distribution channel in a few years. 

Apps serve many purposes, but when property or even lives are on the line, what good are broken tools?  

What is the plan for this app at this point in time?

**Comment by lukeswitz** [comment]
Okay then, dead project it is.

**Comment by tlaurion** [comment]
@n8fr8 i would advise in doing a broader/separate issue on state of things outside of a closed issue and promote need of maintainership even through FPF.

As of now this project has not good figure.

**Comment by fat-tire** [comment]
This issue made me a little depressed, so I figured I'd try to at least try to bring the build back up.  Didn't really test it, but wasn't sure what the interest was really.    But you end up wtih an apk and the libraries and stuff are all up to latest stable versions.  Doesn't mean it works great or anything... YMMV

  `¯\_(ツ)_/¯`

**Comment by lukeswitz** [comment]
It depresses me you added a Facebook dependency. Different strokes I suppose. Good luck here.

**Comment by fat-tire** [comment]
Uh, iwas a facebook dependency for an _existing_ facebook dependency:

```diff
-    implementation 'com.facebook.fresco:fresco:2.0.0'
+    implementation 'com.facebook.fresco:fresco:2.6.0'
+    implementation "com.facebook.infer.annotation:infer-annotation:0.18.0"
```
([MIT-licensed source](https://github.com/facebook/infer))

You *did* know there was a facebook library [here already](https://github.com/guardianproject/haven/blob/master/build.gradle#L173), right?  My addition is only used to suppress a bunch of warnings during the build.  It still runs without it, but the build is noisier.

**Comment by lukeswitz** [comment]
Yes. Whole app is dependent on other parties. Why it’s been that way since launch is concerning at best for an app most literally based in the fundamentals of privacy; both physical & digital. 

That one I’m specifically not a supporter of.

**Comment by fat-tire** [comment]
No problem.  It's not necessarily a problem to have dependencies for open source projects if they can be verified/audited just like any other source...   Maybe too little too late-- maybe not.  No idea.  I just figured may as well make it build.

**Comment by lukeswitz** [comment]
No harm there.
